cScripts/functions/logistics/fn_logistics_getContainer.sqf

```diff
@@ -0,0 +1,31 @@
+#include "..\script_component.hpp";
+/*
+ * Author: CPL.Brostrom.A, SPC.Turn.J
+ * This function return a array of content
+ *
+ * Arguments:
+ * 0: Crate type <STRING>
+ *
+ * Return:
+ * ARRAY of items or empty
+ *
+ * Example:
+ * [""] call cScripts_fnc_logistics_getContainer;
+ */
+
+params [
+    ["_key", "", [""]], 
+    ["_keysOnly", false, [false]]
+];
+
+if (_key == "") exitWith {};
+
+private _containerMap = GVAR(DATABASE);
+private _container = _containerMap get _key;
```

Comment:
Not sure if the function can take the `_key` variable due to be being a call else it can be `"The key provided does not exist."` Note the empty exit return `[]` after the 
```suggestion
private _container = _containerMap getOrDefaultCall [_key, {[format["%1 does not exist.", _key], "Logistics"] call FUNC(warning); []}]
```